bluej/lib/chinese/labels

```diff
@@ -21,19 +21,22 @@ debugger.assert.notNull=\u975e null
 debugger.assert.null=null
 debugger.assert.delta=\u53d8\u5316
 
-debugger.inspector.title=BlueJ:  \u67e5\u770b\u5668
-debugger.inspector.object.title=BlueJ:  \u5bf9\u8c61\u67e5\u770b\u5668
-debugger.inspector.class.title=BlueJ:  \u7c7b\u67e5\u770b\u5668
-debugger.inspector.enum.title=BlueJ:  \u679a\u4e3e \u67e5\u770b\u5668
-debugger.inspector.interface.title=BlueJ:  \u63a5\u53e3 \u67e5\u770b\u5668
-debugger.inspector.result.title=BlueJ:  \u65b9\u6cd5\u8fd4\u56de\u503c
+debugger.inspector.title=BlueJ:  \u68c0\u89c6\u5668
+debugger.inspector.object.title=BlueJ:  \u5bf9\u8c61\u68c0\u89c6\u5668
+debugger.inspector.class.title=BlueJ:  \u7c7b\u68c0\u89c6\u5668
```

Comment:
looks like there extra spaces here, at least definitely between \ and u for \u68c0